cypress/integration/shared/variables.ts

```diff
@@ -1,6 +1,7 @@
 export const PULL_SECRET = Cypress.env('PULL_SECRET');
 export const SSH_PUB_KEY = Cypress.env('SSH_PUB_KEY');
 export const CLUSTER_NAME = Cypress.env('CLUSTER_NAME');
+export const CLUSTER_BASE_DOMAIN = Cypress.env('CLUSTER_BASE_DOMAIN');
```

Comment:
Dupplicate of `DNS_DOMAIN_NAME`. I think we should unify all code to either of those two.

Reply:
Good catch. I'll change to DNS_DOMAIN_NAME.